CRM/Custom/Form/Option.php

```diff
@@ -397,7 +397,7 @@ public function postProcess() {
     $oldWeight = NULL;
     if ($this->_id) {
       $customOption->id = $this->_id;
-      CRM_Core_BAO_CustomOption::updateCustomValues($params);
+      CRM_Core_BAO_CustomOption::updateValue($customOption->id, $customOption->value);
```

Comment:
Note this is just shifting the updateValue call from L487 to here to match how the code was originally

Reply:
I'm thinking that was buggy before and having it down below is maybe better. For example suppose line 472 changes `$customOption->value`. Then if the update is here you've already updated existing ones to something that is different from what gets saved in civicrm_option_value.

Having it at the bottom as it is right now calls the function even on new fields, but could wrap in `if ($customOption->id)`

Reply:
ok have moved it back now @demeritcowboy